test/fixtures/unicorn/prefer-dataset.pass.ts

```diff
@@ -0,0 +1,3 @@
+/**
+ * TODO: find good way to test this: element.dataset.unicorn = '';
```

Comment:
Should we add this test or make the TODO expiring?

Reply:
Fixed with: https://github.com/ridedott/eslint-config/pull/319/commits/09dc2a5dc7793d9447eb5b3402b77e064cece216